site/_data/mydoc/mydoc_sidebar.yml

```diff
@@ -74,6 +74,29 @@ entries:
       version: all
       output: web, pdf
 
+  - title: Quarks Cookbook
+    audience: writers, designers
+    platform: all
+    product: all
+    version: all
+    output: web, pdf
+    items:
+    - title: Recipe 1. Hello Quarks
```

Comment:
Should the title be `Hello Quarks!` (i.e., with an exclamation point)? I see that other references to the recipe (e.g., in Recipe 2) include the exclamation.